docs/eventing/channels/channels-crds.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Available Channels CRDs"
+linkTitle: "Channels CRDs"
+weight: 40
+type: "docs"
+---
+
+<!--
+This is a generated file and should not be changed manually. All changes should follow the
+procedure:
+
+1. Update the information in [`channels.yaml`](channels.yaml).
+
+2. Run the generator tool:
+    ```shell
+    go run eventing/channels/generator/main.go
+    ```
+-->
+
+This is a non-exhaustive list of Channels `CRD`s for Knative.
```

Comment:
and also CCPs, ClusterChannelProvisioners?

Reply:
reduced to generic "Channels"